### ISSUE TITLE
Alerting favoris : Reprogrammer les alarmes au boot

### ISSUE THREAD

**Issue by lbillon** [24HeuresINSA/24h-android-app#189]

**Comment by lbillon** [comment]
Hey @remipichon , y'a -il un moyen de forcer l'update des resources ? ou bien de récupérer la liste des resources ?
Merci !

**Comment by remipichon** [comment]
L'update des ressources est effectué au boot et peut etre provoqué n'importe ou avec 
'''
        ApplicationVersionServiceImpl.getInstance().checkApplicationVersion(getApplicationContext());
'''

**Comment by lbillon** [comment]
Ça update toutes les resources même si les données sont identiques à celles
du serveur ?

Laurent.

--
Laurent BILLON
06 32 57 58 24

**Comment by remipichon** [comment]
Non, c'est le server qui envoie un 200 avec les données ou un 204 sans données. Si tu veux forcer la mise à jour des données il faut appeler 
````
                dataBackendService.getResourcesAsyncFromBackend(retrofitService, dataVersion, dayResourceArrayList, nightResourceArrayList);
````

avec 
````
dataVersion = "0"
````

**Comment by remipichon** [comment]
Quel est le besoin ? tu as toutes tes infos ?

**Comment by lbillon** [comment]
Le truc c'est que le getResourcesAsyncFromBackend il me faut tout plein d'objets pour l'appeler.

Cette fonction est pas critique, a mon avis on peut la descoper.

Quand tu  dis que "L'update des ressources est effectué au boot"  tu entend le boot du téléphone ou au démarrage de l'appli ?

**Comment by remipichon** [comment]
@lbillon il faudrait qu'on s'appelle pour cette issue
